packages/api/src/brevo-contact/brevo-contact.resolver.ts

```diff
@@ -92,6 +95,39 @@ export function createBrevoContactResolver({
             return new PaginatedBrevoContacts(contacts, count, { offset, limit });
         }
 
+        @Query(() => PaginatedBrevoContacts)
+        async brevoTestContacts(@Args() { offset, limit, email, targetGroupId, scope }: BrevoContactsArgs): Promise<PaginatedBrevoContacts> {
```

Comment:
You can remove the `targetGroupId` here -> this query can only be used to query the test list. if you set a   `targetGroupId` in the args that is not the test list id this will not return anything anyways